lambda/script.py

```diff
@@ -80,25 +81,30 @@ def build_group_dict(api: str, api_version: str, scope: str) -> Dict[str, str]:
     nextPageToken = None
 
     while True:
-        groups = (
-            google_client.groups()
-            .list(
-                pageToken=nextPageToken,
-                domain="digital.cabinet-office.gov.uk",
-                maxResults=200,
+        try:
+            groups = (
+                google_client.groups()
+                .list(
+                    pageToken=nextPageToken,
+                    domain=os.environ["DOMAIN"],
+                    maxResults=200,
+                )
+                .execute()
             )
-            .execute()
-        )
-        if groups:
-            for g in groups["groups"]:
-                group_names = g["name"]
-                group_id = g["email"]
-                group_ids[group_names] = group_id
-
-            if "nextPageToken" not in groups:
-                break
-            else:
-                nextPageToken = groups["nextPageToken"]
+            if groups:
+                for g in groups["groups"]:
+                    group_names = g["name"]
+                    group_id = g["email"]
+                    group_ids[group_names] = group_id
+
+                if "nextPageToken" not in groups:
+                    break
+                else:
+                    nextPageToken = groups["nextPageToken"]
+
+        except HttpError as e:
+            print(f"Http error getting group ids: {e}")
+            continue
```

Comment:
Can these really be `continue`d?

Reply:
you're right, these `continue`s may be doing absolutely nothing now I think of it - I just wanted to ensure it went to the next round of the for loop